src/AppBundle/Entity/Repository.php

```diff
@@ -288,4 +296,36 @@ public function addManifest(Manifest $manifest)
 
         return $this;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getRepositoryStars()
+    {
+        return $this->repositoryStars;
+    }
+
+    /**
+     * @param mixed $repositoryStars
+     *
+     * @return $this
+     */
+    public function setRepositoryStars($repositoryStars)
```

Comment:
Methods are not needed yet but property is. In order to validate Doctrine Schema, you have to add reverse side of the ManyToOne relation : `The association AppBundle\Entity\RepositoryStar#repository refers to the inverse side field AppBundle\Entity\Repository#repositoryStars which does not exist.`

In conclusion, I think we must keep property and `getRepositoryStars` method (to be able to list users who "like" the repo).